tests/D2L.CodeStyle.Analyzers.Test/Pinning/PinnedTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Collections.Immutable;
+using D2L.CodeStyle.Analyzers.Pinning;
+using Microsoft.CodeAnalysis;
+using NUnit.Framework;
+
+namespace D2L.CodeStyle.Analyzers.Tests.Pinning {

```

Comment:
Could these be done as spec tests instead? See the "spec" folder in this project (spec = specification 🤷 )

Reply:
Is there documentation on them?

Reply:
I've moved the analyzer to it's own branch and implemented spec tests in there. You can preview them if you want. https://github.com/Brightspace/D2L.CodeStyle/compare/main...alamarre/add-pinned-analyzer